config/install/key.key.tfa_encryption_key.yml

```diff
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies: {  }
+id: tfa_encryption_key
+label: 'TFA encryption key'
+description: ''
+key_type: encryption
+key_type_settings:
+  key_size: 256
+key_provider: file
```

Comment:
Shouldn't the `key_provider` be `environment`?

Reply:
Hi @anthony-malkoun , yes we had a conversation about it https://digital-engagement.slack.com/archives/C013GKSAE30/p1694676758318889 I will ping devops guys about it.

Reply:
@anthony-malkoun this has been updated to use env variable instead.